sass/themes/sr/2018/components/app-link-block/_app-link-block.scss

```diff
@@ -0,0 +1,216 @@
+// App Link Block
+//
+// Markup: app-link-block.twig
+//
+// Style guide: 20.1003
+
+// remove ios default form styles
+%disable-default {
+  -webkit-appearance: none;
+  border-radius: 0;
+}
+
+// safari only hack
+@media screen and (min-color-index:0) and(-webkit-min-device-pixel-ratio:0) {
+  ::-webkit-input-placeholder {
+    opacity:  0.7;
+  }
+}
+
+::placeholder {
```

Comment:
@deniselashlley, can we move these styles to use the an app link block specific parent selector? Don't really wanna be effecting global styles like this